src/content/en/2021/javascript.md

```diff
@@ -1,20 +1,1543 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: JavaScript
-description: TODO
+description: JavaScript chapter of the 2020 Web Almanac covering the usage of JavaScript on the web, libraries and frameworks, compression, web components, and source maps.
 authors: [NishuGoel]
 reviewers: [soulcorrosion, mgechev, rviscomi, pankajparkar, tunetheweb]
 analysts: [pankajparkar, max-ostapenko, rviscomi]
 editors: [rviscomi, pankajparkar]
 translators: []
 results: https://docs.google.com/spreadsheets/d/1zU9rHpI3nC6jTz3xgN6w13afW7x34xAKBh2IPH-lVxk/
```

Comment:
@NishuGoel could you add a bio for yourself? This is shown at the bottom of the chapter.
```suggestion
results: https://docs.google.com/spreadsheets/d/1zU9rHpI3nC6jTz3xgN6w13afW7x34xAKBh2IPH-lVxk/
NishuGoel_bio: TODO
```